css/styles.css

```diff
@@ -5,7 +5,10 @@
 }
 */
 
```

Comment:
- [ ] kindly remove the empty spaces in the footer section (marked with orange) to match figma design

> Hint: Remove the margins from `.contact-us {
    display: flex;
    align-items: left;
    flex-direction: column;
    margin-left: 18px;
    margin-right: 14px;
    background-color: #f9f9fa;
}` and replace them with paddings



![image](https://user-images.githubusercontent.com/89282221/187531600-ffeedca1-e38c-4e00-9135-6c79ee9581e8.png)